docs/en/multi_modal/internvl.md

```diff
@@ -0,0 +1 @@
+# InternVL
```

Comment:
this file is empty. Will it be updated in another PR?

Reply:
Yes, in another PR. I will mark TODO in it